.github/workflows/auto_cherry_pick.yml

```diff
@@ -63,11 +63,15 @@ jobs:
       - name: Add Parent PR's PRT comment to Auto_Cherry_Picked PR's
         id: add-parent-prt-comment
         if: ${{ always() && steps.cherrypick.outcome == 'success' }}
-        uses: mshick/add-pr-comment@v2
-        with:
-          issue: ${{ steps.cherrypick.outputs.number }}
-          message: ${{ needs.find-the-parent-prt-comment.outputs.prt_comment }}
-          repo-token: ${{ secrets.CHERRYPICK_PAT }}
+        run: |
+          ISSUE_NUMBER=${{ steps.cherrypick.outputs.number }}
+          GITHUB_TOKEN="${{ secrets.CHERRYPICK_PAT }}"
```

Comment:
Not sure! But this must be overriding the default GITHUB_TOKEN comes from Github, if not or anyways its good to rename it just to avoid the confusion !

Reply:
@omkarkhatavkar Can you check this and update PR accordingly?

Reply:
@jyejare , @Gauravtalreja1  we are not overriding the `GITHUB_TOKEN` for this workflow, we are just passing the "${{ secrets.CHERRYPICK_PAT }}" our own Satellie_user token, you know it why we need that if we use the `GITHUB_TOKEN` then the PR would have comment from the GitHubAction user, while we are using `Satellite` user everywhere.

Reply:
@omkarkhatavkar I got that! I just wanted to reiterate that we could have named the `GITHUB_TOKEN` env var to something else for CHERRYPICK_PAT, cause this will actually override the `GITHUB_TOKEN` which in the feature you might want to use in this GHA.

I mean is the naming of env var here deliberate?

Reply:
I can directly pass the CHERRYPICK_PAT instead the variable as well. If that work for you ?

Reply:
Sure! Use it directly :)

Reply:
@jyejare tested here https://github.com/omkarkhatavkar/robottelo/actions/runs/6693395618/job/18184437416?pr=158, token is not `****` looks okay to me

Reply:
Sure!